pkg/rpc/request/param.go

```diff
@@ -154,6 +155,42 @@ func (p Param) GetBytesHex() ([]byte, error) {
 	return hex.DecodeString(s)
 }
 
+// GetCosigners returns a slice of transaction.Cosigner with global scope from
+// array of Uint160 or array of serialized transaction.Cosigner stored in the
+// parameter.
+func (p Param) GetCosigners() ([]transaction.Cosigner, error) {
+	hashes, err := p.GetArray()
+	if err != nil {
+		return nil, err
+	}
+	cosigners := make([]transaction.Cosigner, len(hashes))
+	// try to extract hashes first
+	for i, h := range hashes {
+		var u util.Uint160
+		u, err = h.GetUint160FromHex()
+		if err != nil {
+			break
+		}
+		cosigners[i] = transaction.Cosigner{
+			Account: u,
+			Scopes:  transaction.Global,
+		}
+	}
+	if err != nil {
+		for i, h := range hashes {
+			c, err := h.GetString()
```

Comment:
It looks more like
```
["{\"account\": \"0xcadb3dc2faa3ef14a13b619c9a43124755aa2569\",\"scopes\": \"CalledByEntry\"}"]
```
instead of
```
[
      {
        "account": "0xcadb3dc2faa3ef14a13b619c9a43124755aa2569",
        "scopes": "CalledByEntry"
      }
    ]
```

Reply:
Finally did it! Had to add one more Parameter.